Firebase/Messaging/FIRMessagingRemoteNotificationsProxy.m

```diff
@@ -466,39 +389,36 @@ id getNamedPropertyFromObject(id object, NSString *propertyName, Class klass) {
   return property;
 }
 
-#pragma mark - Swizzled Methods
+#pragma mark - UIApplicationDelegate
```

Comment:
@tejasd Is this required that if I use GULAppDelegateSwizzler and I need to swizzle some methods under UIApplicationDelegate, my class must follow the protocol and implement the methods here?

Reply:
With the current design, yes. If that's a blocker we can create our own protocol, say GULAppDelegate which can be platform agnostic.